lib/improver/nbhood.py

```diff
@@ -402,7 +434,9 @@ def __init__(self, neighbourhood_method, radii_in_km, lead_times=None,
             If True, use a circle with constant weighting.
             If False, use a circle for neighbourhood kernel with
             weighting decreasing with radius.
-
+        ens_factor : float
+            The factor with which to multiple the adjustment to the
+            radii_in_km for more than one ensemble member.
```

Comment:
This definition ought to be the same as the one in Utilities.adjust_nsize_for_ens